server/types/Api.ts

```diff
@@ -11,6 +11,7 @@ export interface KiboIncomingMessage extends IncomingMessage {
   searchParams?: Record<string, string>
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   body?: any
+  originalUrl?: string
```

Comment:
What is the originalUrl property for?

Reply:
As pre-commit hook blocking to commit the code due to "req. originally" used, but didn't declared in the API "KiboIncomingMessage" so code was added.